docs/api/plugin/panel.md

```diff
@@ -0,0 +1,120 @@
+# Panel
```

Comment:
Here also some help would be needed to add an explanation on the different attributes.